src/main/java/gregtech/mixins/mui2/LangKeyMixin.java

```diff
@@ -3,22 +3,13 @@
 import com.cleanroommc.modularui.drawable.text.BaseKey;
 import com.cleanroommc.modularui.drawable.text.LangKey;
 import com.llamalad7.mixinextras.injector.ModifyExpressionValue;
-import org.jetbrains.annotations.NotNull;
 import org.spongepowered.asm.mixin.Mixin;
-import org.spongepowered.asm.mixin.Shadow;
 import org.spongepowered.asm.mixin.injection.At;
-import org.spongepowered.asm.mixin.injection.Inject;
-import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
-
-import java.util.function.Supplier;
 
 // all this mixin does is switch newlines to the expected format
```

Comment:
Honestly, Does this mixin even work? I am still getting the different facing newlines in old GUIs, see Phantom Slot in ME Stocking Input Bus